addons/point_of_sale/static/src/js/ChromeWidgets/CashierName.js

```diff
@@ -14,6 +14,14 @@ odoo.define('point_of_sale.CashierName', function(require) {
                 return '';
             }
         }
+        get avatar() {
+            const cashier = this.env.pos.get_cashier();
+            if (cashier) {
+                return `/web/image/res.users/${cashier.user_id[0]}/avatar_128`;
+            } else {
```

Comment:
this.db.get_cashier() || this.get('cashier') || this.employee, '' seems not possible..